peppy/project.py

```diff
@@ -205,8 +206,10 @@ def from_dict(self, pep_dictionary: dict) -> "Project":
         """
         _LOGGER.info(f"Processing project from dictionary...")
 
+        pep_dictionary = {k.lstrip("_"): v for (k, v) in pep_dictionary.items()}
```

Comment:
Is this needed now that we are removing all `_key`'s with `key`?

Reply:
Yes, because in pephub database all keys are with underscore (・へ・)

Reply:
I know this is addressing issues with FastAPI. Would it make more sense to move this logic to the pephub server instead so that we aren't messing with core `peppy` stuff and potentially breaking things? Not sure.

I guess we could also just run a script on the database that does this conversion too so the PEPs there are compatible with the new `peppy`.